format/tmux/formater_test.go

```diff
@@ -58,6 +60,7 @@ func TestFormater_flags(t *testing.T) {
 			symbols: symbols{
 				Conflict:  "SymbolConflict",
 				Untracked: "SymbolUntracked",
+				Delimiter: " - ",
```

Comment:
I'd like to see at least a test case with another delimiter than the default one, to check we're actually showing the strings from the config.
Plus, as said in the issue, I'd like to see at least a test where we expect the components to be shown in reverse order with respect to the default

Reply:
The tests are more of an integration level test.  They may be some what fragile due to the nature of git branch naming.  I believe I have covered most naming conventions.  Also had to get a little creative to get travis CI to work.  It doesn't ever get remote branch data.  Let me know what you think.